src/pages/index.js

```diff
@@ -116,9 +129,7 @@ profileEditBtn.addEventListener("click", () => {
 
 const profilePictureModal = new PopupWithForm(
   "#profile-picture-edit-modal",
-  () => {
-    const values = profilePictureModal.getInputValues();
```

Comment:
**Could be improved**

So, now `getInputValues` can again be `_getInputValues` (**private**) as you wanted in the 8th sprint